<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Midias</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LblTitulo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Que Mídia deseja Adicionar?</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="OpcLivro">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Livros</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="OpcFilme">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Filmes</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="OpcMusica">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>89</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Música</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="BtnAdicionar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+ Adicionar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnListar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Listar Mídias</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TxtValor1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TxtValor2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TxtValor3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="TxtResultado">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
